wix/Build.OpenJDK_generic.cmd

```diff
@@ -147,6 +120,8 @@ FOR %%A IN (%ARCH%) DO (
     IF !PLATFORM! == x86-32 (
         SET PLATFORM=x86
     )
+    @REM Microsoft uses "x86" instead of "x86-32" for the platform folder
+    IF !VENDOR! == Microsoft SET FOLDER_PLATFORM=!PLATFORM!
```

Comment:
Is this something we could set as a default property but be able to override inject that property directly instead of having to do the VENDOR check inside here?

Reply:
I could add `x86` as an additional valid platform. Would we prefer that? @karianna @gdams